packages/web-api/package.json

```
@@ -41,8 +41,8 @@
     "coverage": "codecov -F webapi --root=$PWD"
   },
   "dependencies": {
-    "@slack/logger": "^1.0.0",
-    "@slack/types": "^1.1.0",
+    "@slack/logger": "^2.0.0",
```

```suggestion
    "@slack/logger": ">=1.0.0 <3.0.0",
```

same reasoning as above ☝️